### DISCUSSION TITLE
뉴월드 포즈나 사진찍기 기능

### DISCUSSION THREAD

**Discussion by uniconnect147** [naverz/zepeto-studio-kor#1044] [5. Q&A]
뉴월드로 맵을 만들어 출시했는데요~ 기존 빌드잇에 포즈나 사진찍기기능이 없더라구요;; 이거 구현하고 싶은데 어떻게 해야하나요???

**Comment by minjujuu** [comment]
https://github.com/naverz/zepeto-studio-kor/discussions/437
공식 샘플을 활용해보실 수 있을 거 같아요!
